config/main.py

```diff
@@ -733,7 +733,9 @@ def _get_sonic_services():
 
 def _get_delayed_sonic_services():
     out = clicommon.run_command("systemctl list-dependencies --plain sonic-delayed.target | sed '1d'", return_cmd=True)
-    return (unit.strip().rstrip('.timer') for unit in out.splitlines())
+    timers = [unit for unit in out.splitlines() if(clicommon.run_command("systemctl is-enabled {}".format(unit),
```

Comment:
>systemctl [](http://example.com/codeflow?start=74&length=9)

you can use one command to check all the units. You may use `xargs` to bypass the command line too long issue.